src/Garn.hs

```diff
@@ -2,24 +2,23 @@
 
 module Garn
   ( Env (..),
-    run,
     readOptionsAndConfig,
     runWith,
+    run,
   )
```

Comment:
I think we might be able to export less items here, since we're not using them in tests anymore.